src/webapp/src/client/app/lotteries/lotteries-widget/lotteries-detail-modal.directive.js

```diff
@@ -90,7 +90,13 @@
         function buyTickets(){
             if(vm.ticketCount > 0)
             {
-                vm.notifySrv.success(vm.localeSrv.formatTranslation('lotteries.hasBeenBought', { one: vm.ticketCount, two: vm.lottery.title }));
+                var lotteryTickets = {};
+                lotteryTickets.lotteryId = currentLottery;
+                lotteryTickets.tickets = vm.ticketCount;
```

Comment:
initialize in one liner:

const lotteryTickets = { lotteryId: currentLoterry, tickets: vm.ticketCount };